ConsoleApp/Generics/Stack.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+namespace ConsoleApp;
+
+class Stack<T>
+{
+    private record StackItem
+    {
+        public T Value { get; init; }
+        public StackItem Previous { get; set; }
+
+    }
+    private StackItem _head;
+    public int Length { get; private set; }
```

Comment:
it should be Count instead
```suggestion
    public int Count { get; private set; }
```